kse/src/org/kse/crypto/x509/ExtendedKeyUsageType.java

```diff
@@ -77,8 +77,11 @@
 	NETSCAPE_SGC("2.16.840.1.113730.4.1",  "NetscapeSGCExtKeyUsage"),
 
 	/** Microsoft Server Gated Crypto (SGC) */
-	VERISIGN_SGC("2.16.840.1.113733.1.8.1",  "VeriSignSGCExtKeyUsage");
+	VERISIGN_SGC("2.16.840.1.113733.1.8.1",  "VeriSignSGCExtKeyUsage"),
 
+	/** TSL signing specified by ETSI TS 102 231 V3.1.2 for the purpose of signing Trust-service Status Lists. */
+	TSL_SIGNING("0.4.0.2231.3.0",  "TSLSignExtKeyUsage");
```

Comment:
You forgot to add an entry to the resource bundle "org/kse/crypto/x509/resources" with key TSLSignExtKeyUsage. When you try to view a certificate with this EKU, an MissingResourceException is thrown.